pmml/pmml.py

```diff
@@ -31,159 +29,98 @@ def get_threshold(node_id, prev_node_idx, is_child):
     return threshold[idx]
 
 
-def print_simple_predicate(
-        tab_length,
-        node_id,
-        is_left_child,
-        prev_node_idx,
-        is_leaf,
-        pmml_out):
+def print_simple_predicate(tab_len, node_id, is_left_child, prev_node_idx, is_leaf):
     if is_left_child:
         op = 'equal' if decision_type[prev_node_idx] == 1 else 'lessOrEqual'
     else:
         op = 'notEqual' if decision_type[prev_node_idx] == 1 else 'greaterThan'
-    print('\t' * (tab_length + 1) + ("<SimplePredicate field=\"{0}\" " + " operator=\"{1}\" value=\"{2}\" />") .format(
-        get_field_name(node_id, prev_node_idx, is_leaf), op, get_threshold(node_id, prev_node_idx, is_leaf)), file=pmml_out)
+    out_('\t' * (tab_len + 1) + ("<SimplePredicate field=\"{0}\" " + " operator=\"{1}\" value=\"{2}\" />").format(
+        get_field_name(node_id, prev_node_idx, is_leaf), op, get_threshold(node_id, prev_node_idx, is_leaf)))
 
 
-def print_nodes_pmml(**kwargs):
-    node_id = kwargs['node_id']
-    pmml_out = kwargs['out_file']
-    tab_len = kwargs['tab_length']
+def print_nodes_pmml(node_id, tab_len, is_left_child, prev_node_idx):
     if node_id < 0:
-        node_id = -1 * node_id
-        score = leaf_value[node_id - 1]
-        recordCount = leaf_count[node_id - 1]
+        node_id = ~node_id
+        score = leaf_value[node_id]
+        recordCount = leaf_count[node_id]
```

Comment:
I think `node_id = ~ node_id` should be same as `node_id = - node_id - 1` before? strange thing the model I generate seems a little different with this change